sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/NonIdempotentWriteRetryOptions.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos;
+
+/**
+ * Class to define the options for non-idempotent write operations
+ */
+public final class NonIdempotentWriteRetryOptions {
```

Comment:
Most of our `options` classes are in `com.azure.cosmos.models` package. Any specific reason we are putting this outside?

Reply:
Those are request options, correct? Everything used to configure ClientBuilder (SessionRetryOptions, CosmosDIagnosticsHandler, ThrottlingRetryOptions, ThroughputControl settings are all int he main package. Leaving those here.